File: LabSheet_7.2.c
// C Program to check whether a number is positive or negative or zero using 'if' and 'else' statements

#include <stdio.h>
int main()
{
    int num;
    printf("Enter a number : ");
    scanf("%d", &num);
    if (num > 0)
    {
        printf("%d is an positive number.", num);
    }
    else if (num < 0)
    {
        printf("%d is an negative number.", num);
    }
    else
    {
        printf("The number is zero.");
    }
    return 0;
}

File: Semester 2 - Programming in Python/Labsheet 07/labsheet_7.2.py
# Python Program to get Fibonacci series upto n terms, and:
# i. Count the number of even and odd numbers in the series
# ii. Find the largest Fibonacci number generated
# iii. Compute the sum of the Fibonacci series

def Fibonacci(n):
    if n <= 0:
        print("Fibonacci series can't be found for negative n terms!")
        return

    a, b = 0, 1
    evenCount, oddCount, total = 0, 0, 0

    print("Fibonacci series:")
    for i in range(n):
        print(a, end=" ")
        total += a

        if a % 2 == 0:
            evenCount += 1
        else:
            oddCount += 1

        a, b = b, a + b

    print(f"\n\nTotal number of elements in the series: {n}")
    print(f"Sum of the Fibonacci series: {total}")
    print(f"Even Count: {evenCount}")
    print(f"Odd Count: {oddCount}")
    print(f"Largest Fibonacci number in the series: {b-a}")

num = int(input("Enter the number of terms: "))
Fibonacci(num)